### PR TITLE
Contact Journal screenshots(EXPOSUREAPP-4559)

### PR DESCRIPTION
**Pull request by mtwalli**
Covers contact journal (Screenshot) tests.

### Testing
1. Have Fastlane installed 
2. run `./gradlew clean assembleDebug assembleAndroidTest` to build two APKs. It is required, please don't use APK build using AS
3. run `bundle exec fastlane screengrab`  or `bash screenshot.sh`. the later script will just ZIP screenshots after taking them.

### CODE REVIEW

**Review by chris-cwa — DISMISSED** [review]
⭐️⭐️⭐️⭐️⭐️
Worked perfectly on Pixel 3a / Android 10.

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/ContactDiaryDayFragmentTest.kt
Comment (ralfgehrer) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/contactdiary/ContactDiaryDayFragmentTest.kt:85]: a very minor thing for the future: in unit tests, we have the convention of naming tests as follows
`fun `launch fragment()``

**Comment by ralfgehrer** [comment]
LGTM. Successfully tested on Pixel (Android 10)

Solely waiting for UX mock data approval.

**Review by ralfgehrer — APPROVED** [review]

**Review by chris-cwa — APPROVED** [review]

**Pull request merged**
